### DISCUSSION TITLE
How/Can I specify the Chromium version for Playwright "test-storybook"

### DISCUSSION THREAD

**Discussion by alexbjorlig** [storybookjs/storybook#25698] [Help]
### Summary

I'm trying to setup CI tests for Storybook, and have a setup that already uses Playwright.

Because of this, I already have a version of Chrome installed in my project `chromium-1071`.

How can I instruct Storybook to use this version?

If I don't do anything, I can see storybook is looking for version `chromium-1091`.

Thanks 👋

![CleanShot 2024-01-22 at 12 37 31@2x](https://github.com/storybookjs/storybook/assets/9967422/d96d695a-9be4-4fc1-b3f0-5953c37668b8)

Workaround : Even if I run `npx playwright install` I still only get one version of chromium in CI...

**Comment by renatewr** [comment]
I am also facing this issue.

In my Github Action, `npx playwright install --with-deps` gives chromium-1067, however when running `test-storybook`, I get this error: `Error: Executable doesn't exist at /home/runner/.cache/ms-playwright/chromium-1112/chrome-linux/chrome`.




<img width="761" alt="Screenshot 2024-04-05 at 10 15 45" src="https://github.com/storybookjs/storybook/assets/881272/1292a345-ae7a-4472-8587-9386569bdd9a">

**Reply by renatewr** [reply]
I found this: https://github.com/microsoft/playwright/issues/29930 and I guess its related.

**Reply by renatewr** [reply]
I manged to solve my problem by making sure all dependencies related to playwright (@playwright/test) was updated

**Reply by renatewr** [reply]
@alexbjorlig I dont know if you have solved your issue already. If not, I would take a look in i.e package.json and package-lock and make sure only one version of playwright is used.
